src/channel.rs

```diff
@@ -434,8 +476,9 @@ pub struct IsoTPSettings {
     ///
     /// NOTE: This value might be overridden by the device's implementation of ISO-TP
     pub st_min: u8,
-    /// Use extended ISO-TP addressing
-    pub extended_addressing: bool,
+    /// Extended addressing bytes
+    /// order is Tx ext address, Rx ext address
+    pub extended_addresses: Option<(u8, u8)>,
```

Comment:
Is this type still matching with the [examle](https://github.com/rnd-ash/ecu_diagnostics/blob/11-software-based-iso-tp-channel/examples/passthru_isotp/src/main.rs#L61) for setup IsoTPSettings?

Reply:
Fixed with 2a0acc9d69dc22e8f0478ca68788fcaba41a9951